webservices/partition/base.py

```diff
@@ -316,6 +277,7 @@ def refresh_child(cls, cycle, queue_old, queue_new):
             # populating the child tables during normal partitioning.
             # Otherwise, an error is thrown due more values being specified
             # than there are columns to accept them.
+            # VM TODO - Is this still needed?
```

Comment:
Why not remove `two_year_transaction_period`from the output of `column_factory`?